### PR TITLE
Design feedback

### PR DESCRIPTION
**Pull request by jpogran**
This adds run status icons and human readable text to the workspace hover and adds [] to the projectName in the description

### CODE REVIEW

File: src/providers/tfc/helpers.ts [hunk 1]
```diff
@@ -46,6 +46,66 @@ export function GetRunStatusIcon(status: string): vscode.ThemeIcon {
...
+    case 'canceled':
+      return 'Canceled';
+    case 'force_canceled':
+      return 'Force canceled';
+    case 'errored':
+      return 'Errored';
+  }
+
+  return '';
+}
```
Comment (radeksimko) [inline src/providers/tfc/helpers.ts:107]: I kind of wish we didn't duplicate this (and used [the same approach as we do for other enums](https://github.com/hashicorp/vscode-terraform/blob/59ec335415d577a6c613864517bac18abb10a7c2/src/terraformCloud/run.ts#L40-L60) already) but I understand the MVP nature of this. 🤷🏻‍♂️

File: src/providers/tfc/workspaceProvider.ts [hunk 2]
```diff
@@ -185,13 +185,18 @@ export class WorkspaceTreeItem extends vscode.TreeItem {
       ? `$(source-control) [${this.attributes['vcs-repo-identifier']}](${this.attributes['vcs-repo']['repository-http-url']})`
       : '';
 
+    const statusIcon = GetRunStatusIcon(lastRun?.status ?? '');
+
+    const statusMsg = GetRunStatusMessage(lastRun?.status ?? '');
```
Comment (radeksimko) [inline src/providers/tfc/workspaceProvider.ts:190]: Given that there can be a workspace without any (last) runs, what's the expected/intended behaviour there?

Comment (jpogran) [reply src/providers/tfc/workspaceProvider.ts:190]: No runs mean no status to report. TFC website does not show a status for workspaces with no runs.

File: src/providers/tfc/helpers.ts (hunk 1 above)
Comment (jpogran) [reply src/providers/tfc/helpers.ts:107]: As well as the MVP nature, if we were to put the ThemeIcon and human readable string into an enum, we'd have to had a ThemeIcon for each item.

File: src/providers/tfc/workspaceProvider.ts (hunk 2 above)
Comment (radeksimko) [reply src/providers/tfc/workspaceProvider.ts:190]: AFAICT the code here would produce empty strings which would below error out because we attempt to access `id` on empty string `$(${statusIcon.id})` wouldn't it?

TL;DR don't we need an extra conditional here somewhere?

Comment (dbanck) [reply src/providers/tfc/workspaceProvider.ts:190]: `GetRunStatusIcon` returns a `new vscode.ThemeIcon('indent')` as fallback, so I think everything should work

Comment (radeksimko) [reply src/providers/tfc/workspaceProvider.ts:190]: True but `GetRunStatusMessage` will return empty string, so below we'd print out `Run Status: \n`, wouldn't we?

Comment (radeksimko) [reply src/providers/tfc/workspaceProvider.ts:190]: Oh, now I see we print out the fallback icon first, then the message - sorry. That's fine then.

**Review by radeksimko — APPROVED** [review]

File: src/providers/tfc/workspaceProvider.ts (hunk 2 above)
Comment (dbanck) [reply src/providers/tfc/workspaceProvider.ts:190]: That's true! We could hide `Run Status:` when the message is empty

**Review by dbanck — APPROVED** [review]
